verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -279,6 +295,14 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
                                                 sex: c.demographics?.sex,
                                                 age: c.demographics?.age,
                                                 country: c.location.country,
+                                                adminArea1:
+                                                    c.location
+                                                        ?.administrativeAreaLevel1 ||
```

Comment:
Does it need the `|| ''` ? The other fields don't do that.

Reply:
on you're right we don't need that